charts/dependency-track/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: dependency-track
-version: 0.8.0
+version: 0.8.1
```

Comment:
This is a breaking change, not a bugfix. I don't think we need to bump the major version as we're still v0, but this should at least be a minor bump.